src/Migration/Migration5_0_0.php

```diff
@@ -74,6 +75,10 @@ public function up(): void
             ])
         );
 
-        Actions::execute($request);
+        try {
+            Actions::execute($request);
+        } catch (Throwable $e) {
+            $this->warning($e->getMessage(), ['action' => PdkBackendActions::UPDATE_ACCOUNT]);
```

Comment:
Liever een vast tekstje als message in de log en dan `[ `e` => $e->getMessage() ]` in de context, je ziet zo niet meer terug dat dit een error tijdens de migratie was